future-apps/payroll/contracts/Payroll.sol

```diff
@@ -840,6 +844,17 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
         revert(ERROR_INVALID_PAYMENT_TYPE);
     }
 
+    function _getTotalOwedCappedSalary(uint256 _employeeId) private view returns (uint256) {
```

Comment:
I think this is a good candidate to make `internal` 🤷‍♂

Reply:
let's make it internal 😄